deltacat/experimental/daft/deltacat_daft_patch.py

```diff
@@ -0,0 +1,327 @@
+        # This is a simplified version, would need to be enhanced for production
+        deltacat_schema = self._convert_schema_to_deltacat(source)
+
+        # Create the table in DeltaCAT
+        table_def = deltacat.create_table(
+            name=name,
+            namespace=namespace,
+            version=version,
+            schema=deltacat_schema,
+            catalog=self._name,
+            **kwargs,
+        )
+
+        return DeltaCATTable._from_obj(table_def)
+
+    ###
+    # drop_*
+    ###
+
+    def drop_namespace(self, identifier: Identifier | str):
+        raise NotImplementedError()
+
+    def drop_table(self, identifier: Identifier | str):
+        raise NotImplementedError()
+
+    ###
+    # get_*
+    ###
+
+    def get_table(self, identifier: Identifier | str, **kwargs) -> Table:
+        namespace, table, version = self._extract_namespace_name_version(identifier)
+
+        # TODO validate this works with iceberg if stream format not set
+        table_def = deltacat.get_table(
+            name=table,
+            namespace=namespace,
+            table_version=version,
+            catalog=self._name,
+            **kwargs,
+        )
+
+        if not table_def:
+            raise ValueError(f"Table {identifier} not found")
+
+        return DeltaCATTable._from_obj(table_def)
+
+    ###
+    # list_*
+    ###
+
+    def list_namespaces(self, pattern: str | None = None) -> list[Identifier]:
+        raise NotImplementedError("Not implemented")
+
+    def list_tables(self, pattern: str | None = None) -> list[str]:
+        raise NotImplementedError("Not implemented")
+
+    def _extract_namespace_name_version(
+        self, ident: Identifier | str
+    ) -> Tuple[str, str, Optional[str]]:
+        """
+        Extract namespace, name,version from identifier
+
+        Returns a 3-tuple. If no namespace is provided, uses DeltaCAT defualt namespace
+        """
+        default_namespace = DEFAULT_NAMESPACE
+        if isinstance(ident, Identifier):
+            if len(ident) == 1:
+                return (default_namespace, ident[0], None)
+            elif len(ident) == 2:
+                return (ident[0], ident[1], None)
+            elif len(ident) == 3:
+                return (ident[0], ident[1], ident[2])
+            else:
+                raise ValueError(
+                    f"Expected table identifier to be in format (table) or (namespace, table)"
+                    f"or (namespace, table, version). Found: {ident}"
+                )
+        elif isinstance(ident, str):
+            # TODO (mccember) implement once format of string confirmed with Daft
+            raise ValueError(
+                "Usage of str identifier for DeltaCAT Daft tables not currently support"
+            )
+        else:
+            raise ValueError(
+                f"Expected identifier for createTable to be either Daft Identifier or str. "
+                f"Provided identifier was class: {type(ident)}"
+            )
```

Comment:
Format confirmed, but you also have a choice. I've been doing the same as iceberg. So here's my common snippet.

```python
if isinstance(identifier, str):
    identifier = Identifier.from_str(identifier)
```

Which calls out to the split on "." implementation. This is much more sane for python people, but you do have the `Identifier.from_sql(input, normalize: bool =False)` as an option.